bitnami/airflow/README.md

```diff
@@ -230,7 +230,7 @@ To make this process easier, the chart contains the `resourcesPreset` values, wh
 
 ### Prometheus metrics
 
-This chart can be integrated with Prometheus by setting `metrics.enabled` true. This will deploy a sidecar container with [airflow-exporter](https://github.com/PBWebMedia/airflow-prometheus-exporter) in all Airflow pods and a `metrics` service, which can be configured under the `metrics.service` section. This `metrics` service will be have the necessary annotations to be automatically scraped by Prometheus.
+This chart can be integrated with Prometheus by setting `metrics.enabled` true. This will configure Airflow components to send StatsD metrics to the StatsD exporter that transform them into Prometheus metrics. The StatsD exporter is deployed as a standalone deployment and service in the same namespace as the Airflow deployment.
```

Comment:
Add link to the upstream project https://github.com/prometheus/statsd_exporter just like we had before

That transformS

Reply:
done at https://github.com/bitnami/charts/pull/30459/commits/d0edc8287060d33f30716114c1ce77d932587731 and https://github.com/bitnami/charts/pull/30459/commits/3a3aaf92de8f25fe840de2f91a6ad4450495e61f